pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -526,6 +526,48 @@ type PrometheusSpec struct {
 	// Defines the runtime reloadable configuration of the timeseries database
 	// (TSDB).
 	TSDB TSDBSpec `json:"tsdb,omitempty"`
+	// TracingConfig configures tracing in Prometheus. This is an experimental feature, it may change in any upcoming release in a breaking way.
+	// +optional
+	TracingConfig *PrometheusTracingConfig `json:"tracingConfig,omitempty"`
+}
+
+type PrometheusTracingConfig struct {
```

Comment:
Done mate!
If I'm not wrong only the bool field is required change to be a pointer.

Reply:
I'm still getting used to the concepts, but I believe that `string` does not have built-in nil, and therefore neither has `Duration`. e.g. you can't compare `somestring == nil`

Reply:
Duration on that context is also a string, the operator is just creating a type named Duration to ensure expressiveness.
I guess built in nil is like if non values provided it will be a default type value like on maps and slices are empty.

But I might be wrong hehe I'm not 100% sure

Reply:
Arthur is correct, since strings don't have a default `nil` value we should declare them as `*string`

Reply:
Amazing @JoaoBraveCoding!
Done 😄